src/app/tests/suites/certification/Test_TC_TSTAT_4_2.yaml

```diff
@@ -0,0 +1,2228 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 42.4.2. [TC-TSTAT-4.2] Preset Test Cases with server as DUT
+
+PICS:
+    - TSTAT.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Thermostat"
+    endpoint: 1
+
+tests:
+    - label: "Step 1: Commission DUT to TH"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label:
+          "Step 2: Test Harness writes to the Presets attribute without calling
+          the StartPresetsSchedulesEditRequest"
+      PICS: TSTAT.S.F08 && TSTAT.S.A0050
+      verification: |
```

Comment:
I did not realize these were manual steps.

We have a requirement today that no new manual tests are acceptable to be checked in. @nivi-apple @woody-apple please revert this file from this PR as a followup PR. we can keep the non-manual tests.